### PR TITLE
Host specific node

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/host-specific-node/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C2%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810022306007994-435%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=QKCUg106G7IeS6BSprvU9ETmP%2F%2Bhz0N%2BC%2F0biNeo2ss%3D&st=2018-10-02T23%3A02%3A13Z&se=2018-11-02T23%3A07%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/toc.yml [hunk 1]
```diff
@@ -438,31 +438,33 @@
   - name: Office Add-in manifest reference
     href: https://docs.microsoft.com/javascript/office/manifest/allowsnapshot?view=office-js
     displayName: Excel, Word, Outlook, PowerPoint, OneNote, Project
-  - name: Office JavaScript API sets overview
+  - name: Office JavaScript APIs overview
```
Comment (AlexJerabek) [inline docs/toc.yml]: Should this be "APIs overview" or "API overviews"? Seriously asking...

File: docs/toc.yml [hunk 2]
```diff
@@ -438,31 +438,33 @@
...
-  - name: Visio-specific JavaScript API overview
-    href: https://docs.microsoft.com/javascript/office/overview/visio-javascript-reference-overview?view=office-js
-    displayName: Visio
-  - name: Word-specific JavaScript API overview
-    href: https://docs.microsoft.com/javascript/office/overview/word-add-ins-reference-overview?view=office-js
-    displayName: Word
-  - name: Common JavaScript API requirment sets
+  - name: Common JavaScript API requirement sets
     href: https://docs.microsoft.com/javascript/office/requirement-sets/office-add-in-requirement-sets?view=office-js
     displayName: PowerPoint, Project
```
Comment (AlexJerabek) [inline docs/toc.yml]: Should this display name include all the hosts?

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/host-specific-node/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C2%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810022358189604-435%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=nhzRlO7Xfwon%2B5xRc7hlBf6d%2B2cnXYRsUK9nfPhd3%2Fo%3D&st=2018-10-02T23%3A54%3A30Z&se=2018-11-02T23%3A59%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**